components/expenses/ExpenseForm.js

```diff
@@ -242,6 +244,9 @@ const ExpenseFormBody = ({
   const stepTwoCompleted = isInvite
     ? true
     : (stepOneCompleted || isCreditCardCharge) && hasBaseFormFieldsCompleted && values.items.length > 0;
+  const isMultiCurrency =
+    hasFeature(collective, FEATURES.MULTI_CURRENCY_EXPENSES) ||
+    hasFeature(collective.host, FEATURES.MULTI_CURRENCY_EXPENSES);
```

Comment:
Just because the collective supports multi-currency does not mean that the expense is multi-currency. Multi-currency is defined by `expense.currency !== collective.currency`.

Reply:
Ah, makes sense. I've added this condition as well.

Reply:
@Betree : I was looking at this further and it seems to me what we need here is, `values.payoutMethod?.data?.currency !== collective?.currency`. Since using `expense.currency` assumes that `expense.currency` exists. This will exist for a already saved (pending expense when we try to edit it) but then `expense.currency` doesn't exist for a new expense. What we should compare I think is what the user entered in the first step of the submit expense flow which is `values.payoutMethod?.data?.currency` vs the `collective.currency`. Does that make sense? 🤔